### PR TITLE
Fix healthcheck

### PR DESCRIPTION
**Pull request by insacuri**
https://trello.com/c/FhwDYnNC/1009-investigate-sdx-gateway-healthcheck

Fixes healthcheck for the rabbit connections and the logging around it.  Previously the pika logging was at error only which reduced noise in the logs but made it so that it was possible to miss out on information.

Also renamed the healthcheck delay to `RABBITMQ_HEALTHCHECK_DELAY` and set it to a more sensible default value.  It's called this instead of `SDX_GATEWAY_HEALTHCHECK_DELAY` as other services might (and should, in the case of seft-consumer-service) want to use the value for their healthchecks if we do more in the future.

## How to test
- Ensure submissions can still flow through (via console)
- Check that sdx-gateway heartbeats in the same way as the other sdx services
- Ensure the rabbit connection task is running (should log lines that look like `2019-07-11T09:54:09.000051Z|INFO: sdx-gateway: 2019-07-11 09:54.09 Rabbit MQ health check response status=ok` every 30 or so seconds?) . There should be 3 of them as it's now checking 3 separate urls

### CODE REVIEW

**Review by AndrewTorrance — DISMISSED** [review]
LGTM , tested with compose locally

**Review by mfcrocker — DISMISSED** [review]

File: app/main.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import os
 
 import tornado
+from tornado import gen
```
Comment (necrophonic) [inline app/main.py]: Can't the two `tornado` imports be on the same line?
```python
from tornado import gen, web
```

File: app/main.py [hunk 2]
```diff
@@ -91,12 +90,10 @@ def process(self, message, tx_id=None):
         try:
             self.publisher.publish_message(message, headers={'tx_id': tx_id})
         except PublishMessageError:
-            logger.exception('Unsuccessful publish. tx_id={}'.format(tx_id))
+            logger.exception('Unsuccessful publish.', tx_id=tx_id)
```
Comment (necrophonic) [inline app/main.py:90]: Is it worth binding `tx_id` to the logger?

**Review by AndrewTorrance — APPROVED** [review]
LGTM

**Review by mfcrocker — APPROVED** [review]

File: app/main.py (hunk 2 above)
Comment (insacuri) [reply app/main.py:90]: I could, but it's only going to affect 2 lines.

**Pull request merged**